Lib/FSW/attitudeControl/controllers_init.m

```diff
@@ -0,0 +1,13 @@
+% Initialization of various control schemes
+%
+% Author: Cole Morgan
+
+controllers = struct;
+
+SlewAndControlConstrained_init; %controller1
```

Comment:
controller1/2/3 hides the intended use of the controller. What if I accidentally use the controller1 parameters (e.g, the gain) for detumble by accident? It should always be obvious that this isn't right - when I merge my changes (to this same file since we both did this) I will change the names to be 'controllers.reorientation', 'controllers.detumble' and 'controllers.tracking' or some variant thereof.